client.py

```diff
@@ -79,7 +81,7 @@ def user_input():  # Will process user inputs entered bu user from command line
                     break
             database.save_changes()
             continue
-        elif user_choice == 'D':
+        elif user_choice == user_menu().get('D', None):
             bmk_name = input("Enter a name for your Bookmark you want to delete:  ")
             database.delete_bookmark(bmk_name)
```

Comment:
how about we leave an empty line at end of the code here
if it is the end of file

otherwise the changes LGTM